pkg/util/paramtable/component_param.go

```diff
@@ -1777,7 +1776,7 @@ Max read concurrency must greater than or equal to 1, and less than or equal to
 	p.MaxGroupNQ = ParamItem{
 		Key:          "queryNode.grouping.maxNQ",
 		Version:      "2.0.0",
-		DefaultValue: "1000",
+		DefaultValue: "100",
```

Comment:
@liliu-z does this fit Knowhere?

Reply:
this is to avoid merge cost latency up.

Reply:
big NQ will introduce big latency in nowadays search arch. Let’s reduce it for now until we can find some other solutions